packages/core/src/dialog/DialogContent.tsx

```diff
@@ -30,16 +30,24 @@ export interface DialogContentProps extends HTMLAttributes<HTMLDivElement> {
 export const DialogContent = forwardRef<HTMLDivElement, DialogContentProps>(
   function DialogContent(props, ref) {
     const { children, className, ...rest } = props;
-    const [scrolled, setScrolled] = useState(false);
+    const [scrollBarTop, setScrollBarTop] = useState(false);
```

Comment:
maybe just me, `scrollBarTop` the name trip me think this is a number

Reply:
will do `scrollledTop` `scrolledBottom`